src/plugins/event-plugins/ResourcePlugin.ts

```diff
@@ -134,10 +133,19 @@ export class ResourcePlugin implements Plugin {
     };
 
     recordResourceEvent = (entryData: PerformanceResourceTiming): void => {
-        if (
-            this.recordEvent &&
-            getHost(entryData.name) !== getHost(this.context.config.endpoint)
-        ) {
+        try {
+            // Ignore monitoring beacons.
+            if (
+                new URL(entryData.name).host ===
+                this.context.config.endpointUrl.host
+            ) {
+                return;
+            }
+        } catch (e) {
+            // Fail-safe ignore
```

Comment:
I don't think we can, because the web client resides in customer applications - one day if we have a way to collect error metrics of web client for ourselves, we could add it here